crates/apollo-compiler/src/parser.rs

```diff
@@ -95,20 +95,6 @@ impl std::fmt::Debug for LineColumn {
     }
 }
 
-/// Parse a schema and executable document from the given source text
-/// containing a mixture of type system definitions and executable definitions.
-/// and validate them.
-/// This is mostly useful for unit tests.
-///
-/// `path` is the filesystem path (or arbitrary string) used in diagnostics
-/// to identify this source file to users.
-pub fn parse_mixed_validate(
```

Comment:
I’ve removed this free function (and its reexport at the crate root) because it’s quite niche: it only makes sense in test cases and “toy” example code. Those can still use the `Parser` method.